doc/electrs-rest-btcd-1.0.0.yaml

```diff
@@ -0,0 +1,936 @@
+openapi: 3.0.0
+tags:
+  - name: address
+    description: Address Operations
+  - name: block
+    description: Block Operations
+  - name: tx
+    description: Transaction Operations 
+  - name: scripthash
+    description: Script Operations
+  - name: mempool
+    description: unconfirmed Block Operations
+  - name: default
+    description: Everything else
+info:
+  title: Electrs Rest API
+  version: 1.0.0
+  description: |
+    A Rust Implementation of the Electrum Server.
+    These docs are for the Standard Bitcoin Network Only.
+servers: 
+  - description: "Blockstream Public (bitcoin)" 
+    url: "https://blockstream.info/api/"
+  - description: "Blockstream Public (liquid)" 
+    url: "https://blockstream.info/liquid/api/"
+paths:
+  /blocks/tip/hash:
+    get:
+      tags:
+        - block
+      summary: Get the hash of the tip block
+      responses:
+        '200':
+          description: The hash of the tip block
+          content:
+            text/plain:
+              schema:
+                type: string
+  /blocks/tip/height:
+    get:
+      tags:
+        - block
+      summary: Get the height of the tip block
+      responses:
+        '200':
+          description: The height of the tip block
+          content:
+            text/plain:
+              schema:
+                type: string
+  /blocks/{start_height}:
+    get:
+      tags:
+        - block
+      summary: Get blocks starting from a specific height
+      parameters:
+        - name: start_height
+          in: path
+          required: true
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: List of blocks
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Block'
+  /block-height/{height}:
+    get:
+      tags:
+        - block
+      summary: Get block header by height
+      parameters:
+        - name: height
+          in: path
+          required: true
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: Block header
+          content:
+            application/json:
+              schema:
+                type: string
+  /block/{hash}:
+    get:
+      tags:
+        - block
+      summary: Returns information about a block
+      description: |
+          Elements-based chains have an additional `proof` field. <!--TODO: add link to block-format -->
```

Comment:
- omit the elements/liquid info (we have a separate spec for liquid)
- omit the html

Reply:
removed